src/lang/extra/korean.txt

```diff
@@ -1291,9 +1294,16 @@ STR_STATION_VIEW_HISTORY_BUTTON                                 :{BLACK}화물
 STR_STATION_VIEW_HISTORY_TOOLTIP                                :{BLACK}대기 화물량 이력을 보여줍니다
 
 STR_DEPARTURES_CAPTION                                          :{WHITE}{STRING} - 실시간 운행 정보
+STR_DEPARTURES_CARGO_MODE_TOOLTIP                               :{BLACK}발착 전광판에 화물 필터 모드 설정
 STR_DEPARTURES_DEPARTURES                                       :{BLACK}{TINY_FONT}발
 STR_DEPARTURES_ARRIVALS                                         :{BLACK}{TINY_FONT}착
-STR_DEPARTURES_VIA_BUTTON                                       :{BLACK}경유
+STR_DEPARTURES_BOTH_COMBINED                                    :혼합
+STR_DEPARTURES_BOTH_SEPARATE                                    :분리
+STR_DEPARTURES_DEPARTURE_MODE_TOOLTIP                           :{BLACK}출발이나 도착을 보여줄지 설정합니다
+STR_DEPARTURES_TIMES_BUTTON                                     :{BLACK}시각
+STR_DEPARTURES_TIMES_TOOLTIP                                    :{BLACK}시간표 상 목적지 도착 시각을 보여줍니다
+STR_DEPARTURES_EMPTY_BUTTON                                     :{BLACK}공차
+STR_DEPARTURES_EMPTY_TOOLTIP                                    :{BLACK}공차로 수송하거나, 화물을 싣고 내리지 않는 정차 역을 포함합니다.STR_DEPARTURES_VIA_BUTTON                                       :{BLACK}경유
```

Comment:
There seems to be two lines combined together here?

Reply:
Didn't noticed, thanks for notice